<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>McuWindow</class>
 <widget class="QWidget" name="McuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_productkey">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>390</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_cmd">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>70</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_42">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>sn：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_37">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>cmd：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_checksum_2">
   <property name="geometry">
    <rect>
     <x>529</x>
     <y>100</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_stop3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>停止位：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="stop_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_data3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>330</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>数据位：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_send3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_39">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>100</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>校验和：</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>331</width>
     <height>261</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="flow">
    <enum>QListView::TopToBottom</enum>
   </property>
   <property name="layoutMode">
    <enum>QListView::SinglePass</enum>
   </property>
   <property name="viewMode">
    <enum>QListView::ListMode</enum>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_analyze">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="datapoint_table">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>30</y>
     <width>341</width>
     <height>531</height>
    </rect>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_send3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>390</y>
     <width>41</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_35">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>product_key:</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_check3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>360</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>校验位：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_open3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>300</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_43">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>大小循环：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_setmode">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_rec3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>接收（F5刷新）：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_44">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>360</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>标识码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rate3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>波特率：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_length">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_sn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_38">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>长度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_wifistate">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>210</y>
     <width>150</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="rate_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>6</number>
   </property>
   <item>
    <property name="text">
     <string>110</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>300</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>38400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>57600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_40">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>220</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>wifi状态：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_productkey">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_updatejson">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>490</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>更新数据点</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_36">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>说明：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="com_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_connect">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>160</y>
     <width>150</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_background2">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>0</y>
     <width>951</width>
     <height>587</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_41">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>130</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>配置方式：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_no3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>串口号：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="check_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>360</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Odd</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Even</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Space</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="data_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>330</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_productkey">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>490</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_upload">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>520</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>上报数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_autoupload">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>520</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>定时上报开</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_45">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>260</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>信号强度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_signal">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>260</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_46">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>300</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>有无连接：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_hasconnect">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>300</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>430</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>430</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_showcmd">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>指令界面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_setting">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>520</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>参数界面</string>
   </property>
  </widget>
  <zorder>label_background2</zorder>
  <zorder>label_productkey</zorder>
  <zorder>label_cmd</zorder>
  <zorder>label_42</zorder>
  <zorder>label_37</zorder>
  <zorder>label_checksum_2</zorder>
  <zorder>label_stop3</zorder>
  <zorder>stop_3</zorder>
  <zorder>label_data3</zorder>
  <zorder>textEdit_send3</zorder>
  <zorder>label_39</zorder>
  <zorder>listWidget</zorder>
  <zorder>label_analyze</zorder>
  <zorder>datapoint_table</zorder>
  <zorder>pushButton_send3</zorder>
  <zorder>label_35</zorder>
  <zorder>label_check3</zorder>
  <zorder>pushButton_open3</zorder>
  <zorder>label_43</zorder>
  <zorder>label_setmode</zorder>
  <zorder>label_rec3</zorder>
  <zorder>label_44</zorder>
  <zorder>label_rate3</zorder>
  <zorder>label_length</zorder>
  <zorder>label_sn</zorder>
  <zorder>label_38</zorder>
  <zorder>label_wifistate</zorder>
  <zorder>rate_3</zorder>
  <zorder>label_40</zorder>
  <zorder>lineEdit_productkey</zorder>
  <zorder>pushButton_updatejson</zorder>
  <zorder>label_36</zorder>
  <zorder>com_3</zorder>
  <zorder>label_connect</zorder>
  <zorder>label_41</zorder>
  <zorder>label_no3</zorder>
  <zorder>check_3</zorder>
  <zorder>data_3</zorder>
  <zorder>pushButton_productkey</zorder>
  <zorder>pushButton_upload</zorder>
  <zorder>pushButton_autoupload</zorder>
  <zorder>label_45</zorder>
  <zorder>label_signal</zorder>
  <zorder>label_46</zorder>
  <zorder>label_hasconnect</zorder>
  <zorder>label_2</zorder>
  <zorder>label_time</zorder>
  <zorder>pushButton_showcmd</zorder>
  <zorder>pushButton_setting</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
